Rdmp.Dicom/PipelineComponents/PrimaryKeyCollisionIsolationMutilation.cs

```diff
@@ -464,14 +464,8 @@ private string GetPostgreSqlDeleteCommand(TableInfo toDelete,string deleteOnColu
 
                 var usingsStr = string.Join(",",usings.Except(new []{toDelete}).Select(t=>syntax.EnsureWrapped(t.GetRuntimeName(LoadBubble.Raw,_namer))));
 
-                return string.Format("DELETE FROM {0} {1} WHERE {2} {3} = @val", 
```

Comment:
To me the original was more readable,  I think if there are a lot of variables and you want to communicate the core SQL outline being run up front string.Format is ok.

Reply:
Changed to be more readable in current PR version